pkgs/top-level/config.nix

```diff
@@ -141,7 +141,7 @@ let
     };
 
     showDerivationWarnings = mkOption {
-      type = types.listOf (types.enum [ "maintainerless" ]);
+      type = types.listOf (types.enum [ "maintainerless" "unstructured-arrays" ]);
       default = [];
```

Comment:
We could default this to on?

Reply:
Ah yes, I should have read beyond `type = ...` 🤦🏻